src/edu/ucsb/cs56/projects/games/states_quiz/QuestionManager.java

```diff
@@ -204,18 +298,49 @@ public boolean receiveAnswer(JButton answerButton) {
             }
 
             randStateIndexes.remove(randIndex);
-            if (!randStateIndexes.isEmpty()) {
-                randIndex = (int) (Math.random() * (randStateIndexes.size() - 1));
-                currentQuestion = randStateIndexes.get(randIndex);
-            }
+	   
+	    if (this.getDifficulty().equals("Easy"))
```

Comment:
Two things about this big block of `if`s:
 - They all look the same.  Factor out the common code!
 - The indentation got weird.